taqueria-config.ts

```diff
@@ -1,55 +1,42 @@
-import type {Config, Task, InstalledPlugin, ConfigArgs} from './taqueria-protocol/taqueria-protocol-types.ts'
-import type {Future, TaqError} from './taqueria-utils/taqueria-utils-types.ts'
-import {ConfigDir, i18n} from './taqueria-types.ts'
-import {SanitizedPath, SanitizedAbsPath, SHA256} from './taqueria-utils/taqueria-utils-types.ts'
-import {debug, readJsonFile, writeJsonFile, joinPaths} from './taqueria-utils/taqueria-utils.ts'
+import * as Config from "@taqueria/protocol/Config"
+import * as Task from "@taqueria/protocol/Task"
+import * as InstalledPlugin from "@taqueria/protocol/InstalledPlugin"
+import * as SanitizedAbsPath from "@taqueria/protocol/SanitizedAbsPath"
+import * as SHA256 from "@taqueria/protocol/SHA256"
+import * as LoadedConfig from "@taqueria/protocol/LoadedConfig"
+import * as TaqError from "@taqueria/protocol/TaqError"
+import type {i18n} from '@taqueria/protocol/i18n'
+import {readJsonFile, writeJsonFile, joinPaths, ensureDirExists, eager} from './taqueria-utils/taqueria-utils.ts'
 import {pipe} from "https://deno.land/x/fun@v1.0.0/fns.ts"
-import {resolve, reject, map, chain, mapRej, chainRej, both} from 'https://cdn.jsdelivr.net/gh/fluture-js/Fluture@14.0.0/dist/module.js'
+import {FutureInstance as Future, resolve, reject, attemptP, map, chain, mapRej, chainRej, both} from 'fluture'
 
-export type AddTaskCallback = (task: Task, plugin: InstalledPlugin, handler: (taskArgs: Record<string, unknown>) => Promise<number>) => unknown
+export type AddTaskCallback = (task: Task.t, plugin: InstalledPlugin.t, handler: (taskArgs: Record<string, unknown>) => Promise<number>) => unknown
 
-export const defaultConfig : Config = {
+export const defaultConfig : Config.t = Config.create({
     language: 'en',
-    plugins: [],
     contractsDir: "contracts",
     testsDir: "tests",
     artifactsDir: "artifacts",
     environment: {
         default: "development",
         development: {
             networks: [],
-            sandboxes: ["local"],
-            storage: {
-           }
+            sandboxes: ["local"]
         }
     },
```

Comment:
Let's add a placeholder property for `network` for now:
```
network: { }
```

Reply:
I think it would be more clear to users for us to use the identifier `testnets` rather than `network` for this property